testing/evaluate_tests.py

```diff
@@ -281,12 +281,15 @@ def evaluate_entities_and_schema(
     if verbose:
         logger.setLevel(logging.INFO)
 
-    entities = most_likely_entities = resp["intents"][num_intents]["entities"]
-    observed_entity_dict = {
-        # keep only the most likely hypothesis from Discovery -> first dict in list of dicts returned by Discovery
-        ent["label"]: ent["matches"][0]["value"]
-        for ent in entities  #most_likely_entities
-    }
+    if resp["intents"]:
+        entities = most_likely_entities = resp["intents"][num_intents]["entities"]
```

Comment:
Maybe inline this so it's fewer large if/else statements
```suggestion
        entities = most_likely_entities = resp["intents"][num_intents]["entities"] if "intents" in resp else []
```

Reply:
:+1: all for the inline!